content/post/2019/12/2d-platformer-demo-rework/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Official 2D Platformer Demo Rework"
+description: We've finished up reworking the official 2D Platformer demo using our code guidelines. Take a look at what we changed and why.
+author: johnny
+
+date: 2019-12-06T14:34:50+01:00
+
+resources:
+  - src: banner.png
+    name: banner
+
+categories:
+    - news
+tags:
+    - gdquest
+    - godot engine
+    - learn godot
+---
+
+We’ve finished making changes to the **official 2D platformer demo**! 
+
+This is a proof of concept project and we'd like to give other demos found in the [official demo repository](https://github.com/godotengine/godot-demo-projects) a similar treatment. This demo felt like a great place to start and show how the [guidelines written by the team](https://www.gdquest.com/open-source/guidelines/godot-gdscript/) can be used.
+
+![Remake of the official Godot 2D platformer][image-platformer]
```

Comment:
If you have a one-time link like this better off just linking directly the file: `![Remake of the official Godot 2D platformer][./platformer-2d.png]` - easier for us to review as we don't have to jump to end of file.

Reply:
When I tried this, it didn't work!

![image-error](https://user-images.githubusercontent.com/3285214/71189230-625f7900-227a-11ea-8226-0078389a372a.png)

Reply:
That's because the syntax is off, the path or url should be in parens: ![alt string](path/to/file)